### PR TITLE
More teams stuff

### PR DESCRIPTION
**Pull request by erfanio**

### CODE REVIEW

File: src/components/Team/details.js [hunk 1]
```diff
@@ -82,12 +82,25 @@ class TeamDetails extends React.Component {
     return (
       <div>
         <Header as="h2">{team.name}</Header>
+        {team.shortDescription}
+
+        <p>
+          {team.devpostLink
+            ? <a href={team.devpostLink}>Team&apos;s devpost link</a>
```
Comment (josh-) [inline src/components/Team/details.js]: `devpost` -> `Devpost`?

Also lets link to https://devpost.com?

File: src/components/Team/details.js [hunk 2]
```diff
@@ -82,12 +82,25 @@ class TeamDetails extends React.Component {
...
+
+        <Header as="h3">Description</Header>
+        {team.longDescription ? team.longDescription : <p>Nothing here yet ;)</p>}
 
         <Header as="h3">Members</Header>
         {members.length > 0 ? members : <p>No members!</p>}
 
         <Header as="h3">Invites</Header>
-        {invited.length > 0 ? invited : <p>No invited!</p>}
+        {invited.length > 0 ? invited : <p>No one invited!</p>}
```
Comment (josh-) [inline src/components/Team/details.js]: 👌

File: src/components/Team/details.js (hunk 1 above)
Comment (erfanio) [reply src/components/Team/details.js]: Like 'Please create a [Devpost](https://devpost.com/) team' when they haven't linked it?

Comment (josh-) [reply src/components/Team/details.js]: Excellent!

File: src/components/Team/details.js [hunk 3]
```diff
@@ -82,12 +82,25 @@ class TeamDetails extends React.Component {
     return (
       <div>
         <Header as="h2">{team.name}</Header>
+        {team.shortDescription}
+
+        <p>
+          {team.devpostLink
+            ? <a href={team.devpostLink}>Team&apos;s Devpost link</a>
+            : <p>Please create a <a href="https://devpost.com/">Devpost</a> team and link it here.</p>}
```
Comment (erfanio) [inline src/components/Team/details.js]: @josh- This addresses the team directly.
We need to think of something else when we add judges to the app (they'd be able to access team details page)

Comment (josh-) [reply src/components/Team/details.js]: Yeah sounds like a plan 👍

**Pull request merged**